node/src/accountant/payment_adjuster/non_unit_tests/mod.rs

```diff
@@ -292,140 +251,240 @@ fn make_payable_account(
     }
 }
 
+fn generate_debt_age(gn: &mut ThreadRng, thresholds: &PaymentThresholds) -> u64 {
+    generate_range(
+        gn,
+        thresholds.maturity_threshold_sec,
+        thresholds.maturity_threshold_sec + thresholds.threshold_interval_sec,
+    ) / 2
+}
+
+fn generate_highly_randomized_payable_account_balance(
+    gn: &mut ThreadRng,
+    thresholds: &PaymentThresholds,
+) -> u128 {
+    // This seems overcomplicated, damn. As a result of simple intentions though. I wanted to ensure
+    // occurrence of accounts with balances having different magnitudes in the frame of a single
+    // scenario. This was crucial to me so much that I was ready to write even this piece of code
+    // a bit crazy by look.
+    // This setup worked well to stress the randomness I needed, a lot more significant compared to
+    // what the naked number generator can put for you. Using some nesting, it broke the rigid
+    // pattern and gave an existence to accounts with diverse balances.
+    let mut generate_u128 = || generate_non_zero_usize(gn, 100) as u128;
+
+    let parameter_a = generate_u128();
+    let parameter_b = generate_u128();
+    let parameter_c = generate_u128();
+    let parameter_d = generate_u128();
+    let parameter_e = generate_u128();
+    let parameter_f = generate_u128();
+
+    let mut use_variable_exponent =
+        |parameter: u128, up_to: usize| parameter.pow(generate_non_zero_usize(gn, up_to) as u32);
+
+    let a_b_c_d_e = parameter_a
+        * use_variable_exponent(parameter_b, 2)
+        * use_variable_exponent(parameter_c, 3)
+        * use_variable_exponent(parameter_d, 4)
+        * use_variable_exponent(parameter_e, 5);
+    let addition = (0..6).fold(a_b_c_d_e, |so_far, subtrahend| {
+        if so_far != a_b_c_d_e {
+            so_far
+        } else {
+            if let Some(num) =
+                a_b_c_d_e.checked_sub(use_variable_exponent(parameter_f, 6 - subtrahend))
+            {
+                num
+            } else {
+                so_far
+            }
+        }
+    });
+
+    thresholds.permanent_debt_allowed_gwei as u128 + addition
+}
+
+fn try_make_qualified_payables_by_applied_thresholds(
+    payable_accounts: Vec<PayableAccount>,
+    applied_thresholds: &AppliedThresholds,
+    now: SystemTime,
+) -> Vec<QualifiedPayableAccount> {
+    let payment_inspector = PayableInspector::new(Box::new(PayableThresholdsGaugeReal::default()));
+    match applied_thresholds {
+        AppliedThresholds::Defaulted => try_to_make_guaranteed_qualified_payables(
+            payable_accounts,
+            &PRESERVED_TEST_PAYMENT_THRESHOLDS,
```

Comment:
In our discussion it was pointed out by you that you may use the `PaymentThresholds::default()` here instead of the constant `PRESERVED_TEST_PAYMENT_THRESHOLDS`.